### ISSUE TITLE
Quests from items not being automatically accepted

### ISSUE THREAD

**Issue by Jokaleth** [Questie/Questie#4125]
## Bug description
As the title says, when I pick up an item with a quest attached, such as Avruu's Orb, the quest is not automatically accepted and tracked. It used to be the case where this would automatically happen, but since at least prepatch this is no longer the case.

## Screenshots
![image](https://user-images.githubusercontent.com/87456827/190241812-f4884abc-79d5-41aa-b011-5b65585496b2.png)
![image](https://user-images.githubusercontent.com/87456827/190241824-d7afa294-115b-489b-81aa-98a8c834d5a8.png)

## Questie & WoW version

Questie Version: v7.1.1
WoW Version: 3.4.0.45572

**Comment by BreakBB** [comment]
I am quite sure this never was the case as it would require Questie to interact with the inventory (which we don't and hopefully won't).

It is a nice idea though 👍🏻

**Comment by Jokaleth** [comment]
> I am quite sure this never was the case as it would require Questie to interact with the inventory (which we don't and hopefully won't).
> 
> It is a nice idea though 👍🏻

Ah I see, then that’s my bad. I always thought it was Questie doing it but if that’s not the case then I know exactly what it was.

I had another addon that would automatically open items in bags. This would normally apply to stuff like clams and boxes, but I guess it must’ve also been automatically opening be quest attached to the item, which Questie then automatically accepted. That addon broke on prepatch launching so I removed it.

**Comment by BreakBB** [comment]
That would make sense. If you find that addon doing this, feel free to share it here 👍🏻

**Comment by MyTechnoHunter** [comment]
Doesnt appear to be listed in my curseforge atm but there's an addon called AutoOpenItems
that does that type thing.. opens clams and boxes and whatnot..

**Comment by Jokaleth** [comment]
> Doesnt appear to be listed in my curseforge atm but there's an addon called AutoOpenItems that does that type thing.. opens clams and boxes and whatnot..
> 
> simple enough to update the toc file for wrath, lemme make sure it still works..

Yeah that’s the one. Like you said it hasn’t been updated yet and was broken on prepatch. Let me know when you update it!

**Comment by Yttrium-tYcLief** [comment]
> Yeah that’s the one. Like you said it hasn’t been updated yet and was broken on prepatch. Let me know when you update it!
This is likely the sort of thing that is now disallowed (deleted/removed API calls) in order to help combat botting/multiboxing. Many other addons like AutoFollow got hit as well.
